core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -280,10 +282,27 @@ private[spark] abstract class MapOutputTracker(conf: SparkConf) extends Logging
     }
   }
 
+  protected def supportsContinuousBlockBatchFetch(serializerRelocatable: Boolean): Boolean = {
+    if (!conf.getBoolean("spark.shuffle.continuousBlockBatchFetch", true)) {
```

Comment:
it's only used in core module, we can define it in `org.apache.spark.internal.config`